README.md

```diff
@@ -15,11 +15,30 @@ Please ensure all tasks have a description that includes a clear definition of w
 
 Builds are run automatically out of this repository. Basic workflow is:
 
-- Create a branch and implement your changes
-- The Docker image is built on the branch and as part of the pull request
-- Once the PR is merged, a build will be run and pushed to `latest` and `sudo-latest` from master
-- When a new stable image is ready, create a release with a version number like `0.0.6` and
-this will be pushed to `0.0.6` and `sudo-0.0.6`.
+- Create a branch and implement your changes.
+- The Docker image is built on the branch and as part of the pull request.
```

Comment:
I'm a bit puzzled about the 'as part of the pull request' bit. 
```suggestion
- A Docker image will be built when changes are pushed to the branch.
```

Reply:
There will be other actions triggered with PR, so docker image building is only part of the whole workflow.

Reply:
Is there any situation when it is triggered by the PR but **wouldn't** have already been triggered by the latest commit to the branch the PR was based on? I'm just wondering whether the distinction has an impact.

I probably should stop thinking about this and just make it say:
...'when changes are pushed to the branch *and* when a PR is created.' ?

Reply:
I can cite two situations, where
- the repo was forked and pr was created on the forked repo and to be merged on this repo. 
- there were multiple local commits and the latest commit wasn't on `paths` listed in the workflow, so the workflow wouldn't be triggered when all the commits were pushed on the remote branch.